src/seeders/20200106072811-users-notification-mode.js

```diff
@@ -0,0 +1,26 @@
+const demoPreferenceMode = {
```

Comment:
Is there really a need to generate seeders for this task?

Reply:
Yes because each user we create in seeds needs to have default preferences set. otherwise, it would throw an error when trying to send notifications

Reply:
I don't see the need to generate seeders for preferences and users tables. You can manually generate these data locally. There's no point having this in staging or production environment